homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -54,18 +54,23 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Waze travel time sensor platform."""
-    destination = config.get(CONF_DESTINATION)
-    name = config.get(CONF_NAME)
-    origin = config.get(CONF_ORIGIN)
-    region = config.get(CONF_REGION)
-    incl_filter = config.get(CONF_INCL_FILTER)
-    excl_filter = config.get(CONF_EXCL_FILTER)
-
-    sensor = WazeTravelTime(name, origin, destination, region,
-                            incl_filter, excl_filter)
-
-    add_devices([sensor], True)
-
+    def run_setup(event):
```

Comment:
Why do we need to delay the start of this platform?

This is never called thus the platform is not set up.

Reply:
We don't need to delay the setup of the platform, we just need to delay the call to update. Reason is that we depend on the state of other entities, from origin and destination config, so those entities need to be setup before we call update.

Reply:
And that delay is already there.

Reply:
Yes!